### DISCUSSION TITLE
Error: read ECONNRESET

### DISCUSSION THREAD

**Discussion by 1PV1** [nodejs/help#4987] [Q&A] [answered]
### Version

16.14.2

### Platform

Linux discord-bots 5.15.30-2-pve #1 SMP PVE 5.15.30-3 (Fri, 22 Apr 2022 18:08:27 +0200) x86_64 GNU/Linux

### What steps will reproduce the bug?

I don't really know how to reproduce it, I tested all my code and nothing.

### How often does it reproduce? Is there a required condition?

It happens every 7-9 hours I think.

### What do you see instead?

So, I have a discord bot that runs on node, I tested everything the bot has but I couldn't reproduce the problem, 2-3 days ago I started getting this error, and since then I keep getting it every 7-9 hours, I don't know the exact time but something like this.
```
 Error: read ECONNRESET
     at TCP.onStreamRead (node:internal/stream_base_commons:217:20) {
   errno: -104,
   code: 'ECONNRESET',
   syscall: 'read',
   fatal: true
 }
```

First I thought that it has to do with my node version, I had the version 16.14.2 before the error started coming up, I updated it to 16.17.1 and then the error started appearing, I changed it back to 16.14.2 and I'm still getting it.

**Comment by bnoordhuis** [comment, marked as answer]
Converted to a discussion. ECONNRESET means the other end closed the connection. Nothing unusual, just reconnect.
